src/components/ConfirmationDialog.tsx

```diff
@@ -97,111 +95,446 @@ export default function ConfirmationDialog({
-                  </TabsTrigger>
-                  <TabsTrigger value='json' className='flex-grow'>
+                  </div>
+                </TabsTrigger>
+
+                <TabsTrigger
+                  value='details'
+                  className='flex-1 rounded-md px-3 py-1 text-sm font-medium'
+                >
+                  <div className='flex items-center justify-center'>
+                    <ListCollapseIcon className='h-4 w-4 mr-2' />
+                    <Trans>Details</Trans>
+                  </div>
+                </TabsTrigger>
+
+                <TabsTrigger
+                  value='json'
+                  className='flex-1 rounded-md px-3 py-1 text-sm font-medium'
+                >
+                  <div className='flex items-center justify-center'>
+                    <BracesIcon className='h-4 w-4 mr-2' />
                     <Trans>JSON</Trans>
-                  </TabsTrigger>
-                </TabsList>
-                <TabsContent value='simple'>
-                  {isHighFee && !fee.isZero() && (
-                    <Alert variant='warning' className='my-3'>
-                      <CircleAlert className='h-4 w-4' />
-                      <AlertTitle>
-                        <Trans>High Transaction Fee</Trans>
-                      </AlertTitle>
-                      <AlertDescription>
-                        <Trans>
-                          Fee exceeds recommended maximum of 0.001 {ticker}
-                        </Trans>
-                      </AlertDescription>
-                    </Alert>
-                  )}
-                  <Group label={t`Summary`} icon={BadgePlus}>
-                    <div className='flex flex-col gap-2'>
-                      {!BigNumber(response?.summary.fee || 0).isZero() && (
-                        <Item
-                          badge={t`Fee`}
-                          label={`${formatNumber({
+                  </div>
+                </TabsTrigger>
+              </TabsList>
+            </div>
+
+            <div className='flex-1 relative'>
+              {/* Transaction Summary Tab */}
+              <TabsContent
+                value='summary'
+                className='absolute inset-0 overflow-auto border rounded-md p-4 bg-white dark:bg-neutral-950'
+              >
+                {/* High Fee Warning */}
+                {isHighFee && !fee.isZero() && (
+                  <Alert variant='warning' className='mb-3'>
+                    <AlertCircleIcon className='h-4 w-4' />
+                    <AlertTitle>
+                      <Trans>High Transaction Fee</Trans>
+                    </AlertTitle>
+                    <AlertDescription>
+                      <Trans>
+                        Fee exceeds recommended maximum of 0.001 {ticker}
+                      </Trans>
+                    </AlertDescription>
+                  </Alert>
+                )}
+
+                {/* Fee Display */}
```

Comment:
The fee is very front and center right now, being at the top and having a very large font.
It looks more important than the rest of the TX.

I would suggest putting it a the bottom of the modal, right above the buttons.
This way it's always in the same place and gets out of the way.

Reply:
how's this look?
![image](https://github.com/user-attachments/assets/20a6d427-ca3d-4ef7-be6b-4f57fa50ba3e)